app/templates/signed-in.html

```diff
@@ -9,6 +9,9 @@
         <h1 class="saturn">Congratulations. You are signed in.</h1>
         <p>You can now access protected endpoints.</p>
         <p>To sign out please <a href="{{ url_for('logout') }}" > select here.</a> </p>
+        {% if data.logged_in == "True" %}
```

Comment:
Should we not use the other way of doing this by looking at the session variable? i.e. it has the jwt token ? :-) this might be a better way - as in the future we can make this a template tag bla bla bla bla..... You know what I mean.
Anyway let me know what you think on that.